commands/delete-database.go

```diff
@@ -15,81 +17,176 @@ func init() {
 	reset := cli.Command{
 		Name:      "delete",
 		Aliases:   []string{"delete-database"},
-		Usage:     "Delete an imported database",
+		Usage:     "Delete imported database(s)",
 		ArgsUsage: "<database>",
 		Flags: []cli.Flag{
 			forceFlag,
 			configFlag,
+			allFlag,
+			matchFlag,
+			regexFlag,
+			dryRunFlag,
 		},
-		Action: func(c *cli.Context) error {
-			res := resources.InitResources(c.String("config"))
-			db := c.Args().Get(0)
-			if db == "" {
-				return cli.NewExitError("Specify a database", -1)
-			}
-
-			return deleteDatabase(db, res, c.Bool("force"))
-
-		},
+		Action: deleteDatabase,
 	}
 
 	bootstrapCommands(reset)
 }
 
 //deleteDatabase deletes a target database
-func deleteDatabase(database string, res *resources.Resources, forceFlag bool) error {
+func deleteDatabase(c *cli.Context) error {
+	res := resources.InitResources(c.String("config"))
+
+	// Different command flags
+	match := c.String("match")
+	regex := c.String("regex")
+	bulk := c.Bool("all")
+	force := c.Bool("force")
+	dryRun := c.Bool("dry-run")
+	var names []string
+	var err error
+
```

Comment:
Do you think we should make the flags exclusive? Should we throw an error if an invalid combination is supplied?
What happens if a user runs `rita delete-database -r -m (foo|bar)`? 
In this implementation, the string `(foo|bar)` would be directly matched against the db list.

@ethack is this something worth dealing with?